.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 <!--
 
-Make sure to run `msbuild RoslynAnalyzers.sln /t:pack` and `msbuild FxCopAnalyzers.sln /t:pack` in the repository root when
+Make sure to run `msbuild /t:pack` in the repository root when
```

Comment:
I would recommend to add the `/v:m` to have the minimal verbosity which speeds up the process.

Reply:
```suggestion
Make sure to run `msbuild /t:pack /v:m` in the repository root when
```